tools/cli/workflow.go

```diff
@@ -218,3 +224,71 @@ func newWorkflowCommands() []cli.Command {
 		},
 	}
 }
+
+func newActivityCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:    "complete",
+			Aliases: []string{"comp"},
+			Usage:   "Complete an activity",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagWorkflowIDWithAlias,
+					Usage: "WorkflowID",
+				},
+				cli.StringFlag{
+					Name:  FlagRunIDWithAlias,
+					Usage: "RunID",
+				},
+				cli.StringFlag{
+					Name:  FlagActivityIDWithAlias,
+					Usage: "The activityID to operate on",
+				},
+				cli.StringFlag{
+					Name:  FlagResult,
+					Usage: "result of the activity",
+				},
+				cli.StringFlag{
+					Name:  FlagIdentity,
+					Usage: "identity of the activity result",
```

Comment:
This should be identity of operator (or worker)

Reply:
Makes sense